backend/.gitignore

```diff
@@ -0,0 +1,23 @@
+# dependencies
```

Comment:
I prefer separate two gitignores because they are separate projects. Having different ignore files in separate directories makes it less overwhelming to read. It is more convenient to modify without regarding complex file path.

Reply:
I had to add my own gitignore to the root directory in order to check through the project because the internal gitignores don't cover the ./build and ./node_modules that have 1000's of files in them. I recommend having a root gitignore in addition to the two separate gitignores then so that people don't accidentally commit massive files on their PR's.

Reply:
Git ignore can cover everything inside a folder. /node-module contains more than 10k files. I can only think of some ide will generate cache folder like .vscode in root dir, or some software can generate junk files in root. That maybe the case you mention. I'll add it.